src/main/java/appeng/parts/AEBasePart.java

```diff
@@ -498,4 +495,13 @@ public ISimplifiedBundle getRenderCache() {
     public void setRenderCache(final ISimplifiedBundle renderCache) {
         this.renderCache = renderCache;
     }
+
+    private static int getSide(ForgeDirection direction) {
```

Comment:
Given that there's already a member function called `getSide()`, it might be better to name this something different to avoid confusion, like `getSideIndexFromDirection`. Or put a simple comment on what it's doing (extracting index from ForgeDirection for `PlayerInteractEvent`).